cypress/integration/core-opensearch-dashboards/opensearch-dashboards/apps/query_enhancements/dataset_selector.spec.js

```diff
@@ -2,65 +2,47 @@
  * Copyright OpenSearch Contributors
  * SPDX-License-Identifier: Apache-2.0
  */
-import {
-  MiscUtils,
-  TestFixtureHandler,
-} from '@opensearch-dashboards-test/opensearch-dashboards-test-library';
-import { PATHS, SECONDARY_ENGINE } from '../../../../../utils/constants';
-
-const miscUtils = new MiscUtils(cy);
-const testFixtureHandler = new TestFixtureHandler(cy, PATHS.ENGINE);
-
-describe.skip('dataset selector', { scrollBehavior: false }, () => {
-  describe('empty state', () => {
```

Comment:
why do we delete the test for testing the empty state? i think we do need to test when there is no index pattern

Reply:
we moved it to a_check.spec.js